### DISCUSSION TITLE
arm based processor support

### DISCUSSION THREAD

**Discussion by gbene** [JordanKnott/taskcafe#146] [General]
Hi!
I would like to use taskcafe on my raspberry pi server (arm based architecture) but the required image (armv7) is not present in the docker page. 
For the moment I'm using a fork https://github.com/TanguyHerbron/taskcafe but it is not updated to the latest version (creating labels brings up an empty page as described in https://github.com/JordanKnott/taskcafe/issues/121). Do you have any intention to port taskcafe to arm in the near future? It would be nice to have an always up to date official version for this architecture.     
Thanks!

**Comment by mooujarrar** [comment]
+1
